Sources/VRage.Dedicated/Configurator/ConfigForm.cs

```diff
@@ -69,7 +69,7 @@ private void startButton_Click(object sender, EventArgs e)
             else // Local / Console
             {
                 string[] cmdLine = Environment.GetCommandLineArgs();
-                Process.Start(cmdLine[0], ((cmdLine.Length > 1) ? cmdLine[1] : "" ) + " -console -ignorelastsession");
+                Process.Start(cmdLine[0], ((cmdLine.Length > 1) ? String.Join(" ", cmdLine, 1, cmdLine.Length - 1 ) : "") + " -console -ignorelastsession");
```

Comment:
"+" -console... will concatenate it a second time. I'm just asking if that is ok for the new process."
-console option check by Contains method. so i see no problemo...